soap/src/java/com/zimbra/soap/mail/message/CreateSearchFolderRequest.java

```diff
@@ -34,13 +36,15 @@
  */
 @XmlAccessorType(XmlAccessType.NONE)
 @XmlRootElement(name=MailConstants.E_CREATE_SEARCH_FOLDER_REQUEST)
+@GraphQLType(name=MailConstants.E_CREATE_SEARCH_FOLDER_REQUEST, description="create search folder request")
 public class CreateSearchFolderRequest {
 
     /**
      * @zm-api-field-description New Search Folder specification
      */
     @ZimbraUniqueElement
     @XmlElement(name=MailConstants.E_SEARCH /* search */, required=true)
+    @GraphQLQuery(name="searchFolder", description="name of the search folder")
```

Comment:
Please use constants instead of direct string.

Reply:
We cannot use all of the constants because names like "l" and "search" are not self-explanatory enough for this service. We may need to come up with something to handle this. For now direct string to provide a descriptive name is what should be done.

Reply:
created new class GqlConstants in zm-mailbox-> common